package.json

```diff
@@ -65,7 +68,10 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.10.1",
+    "@types/node-schedule": "^1.3.0",
     "bulma": "^0.8.0",
-    "bulma-extensions": "^6.2.7"
+    "bulma-extensions": "^6.2.7",
+    "node-schedule": "^1.3.2",
+    "parse-reminder": "^1.2.0"
```

Comment:
Do we really need these 3rd party packages? Can't this be implemented without any extra dependencies?

Reply:
ParseReminder: allows to parse expression in to reminders - it simplifies development as it allows to break down the reminder expression to 'what', 'when' and 'where' which can then be used for the notifications
Node-Schedule: allows to schedule events - you can use the date parsereminder date to schedule notifications

Reply:
It should be quite possibly to replace parse reminder with a built-in implementation, as its code is rather simple. Replacing chrono-js, which parse-reminder depends on seems to be a fair bit harder. https://github.com/bkeepers/parse-reminder/blob/master/index.js

Replacing node-schedule with a custom made solution should be rather simple, because it doesn't actually schedule a cronjob or anything. Instead, "scheduled jobs will only fire as long as your script is running".

Reply:
@stefnotch i will look in replacing parse-reminder with a custom one based on it - but having a look at the chrono-js it is beyond my capabilities right now
regarding the node-schedule i think a timeout would work? it would have the same capabilities. How would we go about notification when the code is not running i.e. restart?

Reply:
I suppose it would be acceptable to assume that ueli is always running and only show reminders while ueli is running. Regarding an ueli restart, I think you pretty much have to write the reminders to a file or somehow persist them. But I'm not sure, so I'm pinging @oliverschwendener

Reply:
@stefnotch yeah you are right about a file/sqlite database for the persist. and load them everytime the reminder load. 
Would a calendar integration be better for this?

Reply:
@stefnotch @oliverschwendener i have implemented my own parse reminder. However for the scheduler i am unsure how to do it. Settimout max value is around 24 days. Would this be sufficient?